doc/source/getting_started/notebooks/notebook2python.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
Strongly against platform specific stuff like `.sh`, why not make this script in python?